lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -150,6 +157,40 @@ export class SilentIframeClient extends StandardInteractionClient {
                 (e as AuthError).setCorrelationId(this.correlationId);
                 serverTelemetryManager.cacheFailedRequest(e);
             }
+
+            if (
+                e instanceof ServerError &&
+                e.errorCode === BrowserConstants.INVALID_GRANT_ERROR
+            ) {
+                if (!authClient) {
```

Comment:
Optional. You can simplify nested `if` conditions by joining them together into `if (!authClient || !(e instanceof ServerError) || e.errorCode !== BrowserConstants.INVALID_GRANT_ERROR) { then throw e }` so you won't need `else`.